Games/Chess/creer.yaml

```diff
@@ -31,7 +33,8 @@ AI:
       serverPredefined: true
       returns:
         type: string
-        description: A string in Standard Algebriac Notation (SAN) for the move
-          you want to make. If the move is invalid or not properly formatted
-          you will lose the game.
+        description: A string in UCI (Universal Chess Interface), or SAN
+          (Standard Algebraic Notation) for the move you want to make.
```

Comment:
I would recommend specifying that only the PGA format variation of SAN notation is supported.
Also I feel we are semantically misusing the term 'UCI'. 
UCI is the standardized protocol for an engine to interface with a chess arena, including how to start the engine, configuring engine settings like turn timings, setting up the board state, etc. For communication of moves, UCI uses a variation of the Long Algebraic Notation (LAN) which is simply {from}{to}{promotion}.

Reply:
Ok thank you for clarifying this. I did not know these subtle differences, and will update these docs accordingly.

Reply:
Updated in e1e2d5056c7b722c58a3ab77896c4f7317718f47